cdc/sink/mysql/mysql_params.go

```diff
@@ -304,6 +304,9 @@ func generateDSNByParams(
 	// equals to executing "SET NAMES utf8mb4"
 	dsnCfg.Params["charset"] = defaultCharacterSet
 
+	// disable foreign_key_checks
+	dsnCfg.Params["foreign_key_checks"] = "0"
```

Comment:
Does TiDB 4.0.0 support the variable it?

Reply:
Yes, TiDB 4.0.0 support the variable.